### PR TITLE
uploading header

### PR DESCRIPTION
**Pull request by luchogabutti**
uploading header of my reddit page

Screen:

image

Linear card:

https://linear.app/cuadernorojo/issue/CR-79/header

### CODE REVIEW

File: src/components/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+            <p className='title'>Lucheddit</p>
+          </Typography>
+        </Toolbar>
+      </AppBar>
+    </Box>
+  );
+}
```
Comment (noelsp) [inline src/components/Header.jsx]: falta agregar end of line!

File: src/App.css [hunk 2]
```diff
@@ -1,29 +1,38 @@
...
+  font-weight: bold;
+  margin-left: 7px;
+}
+
+.articles {
+  margin: 10px;
+  padding: 10px;
+  border-radius: 10px;
+  background-color: red;
+  width: 70%;
```
Comment (noelsp) [inline src/App.css:26]: you should use grid system to show articles in the 70% of the page.

File: src/App.js [hunk 3]
```diff
@@ -1,24 +1,62 @@
...
+
+      res.json().then(data => {
+        if (data != null) {
+          setArticles(data.data.children);
+        }
+      })
+
+    });
+
+  }, [subreddit]);
```
Comment (noelsp) [inline src/App.js]: we should move this to another component

File: src/App.js [hunk 4]
```diff
@@ -1,24 +1,62 @@
...
-        >
-          Learn React
-        </a>
-      </header>
+      <Header/>
+      <div className='articles'>
+        {
+          (articles != null) ? articles.map((article, index) => <Article key={index} article={article.data}/>) : ''
+        }
+      </div>
```
Comment (noelsp) [inline src/App.js]: We should move this to another component called "Content"

`(articles != null) ? articles.map((article, index) => <Article key={index} article={article.data}/>) : ''`
--> 
`{articles && articles.map(article => <Article key={article.id} article={article} />`

File: src/App.js [hunk 5]
```diff
@@ -1,24 +1,62 @@
...
+      if (res.status !=200) {
+        console.log('Error');
+        return;
+      }
+
+      res.json().then(data => {
+        if (data != null) {
+          setArticles(data.data.children);
+        }
+      })
```
Comment (noelsp) [inline src/App.js]: if we have "data" inside children, we should do : data.data.children.data to setArticles here.

File: src/components/Article.jsx [hunk 6]
```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+
+function Article(props){
```
Comment (noelsp) [inline src/components/Article.jsx]: We prefer this: 
```
const Article = (props) => {
...
}
```

File: src/App.js [hunk 7]
```diff
@@ -1,24 +1,62 @@
...
+  },
+});
 
 function App() {
+
+  const [articles, setArticles] = useState([]);
+  const [subreddit, setSubreddit] = useState('');
+
+  useEffect(()=> {
+    fetch('https://www.reddit.com/'+subreddit+'.json').then(res=>{
```
Comment (noelsp) [inline src/App.js]: `"https://www.reddit.com/${subreddit}.json"` use comillas francesas

**Review by noelsp — CHANGES_REQUESTED** [review]

File: src/components/Header.jsx (hunk 1 above)
Comment (luchogabutti) [reply src/components/Header.jsx]: corrected!

File: src/App.js (hunk 3 above)
Comment (luchogabutti) [reply src/App.js]: I want to check this with you my friend!

File: src/App.js (hunk 5 above)
Comment (luchogabutti) [reply src/App.js]: I tried to do this but it didnt work so i keep it like this!!!

File: src/components/Article.jsx (hunk 6 above)
Comment (luchogabutti) [reply src/components/Article.jsx]: done brother

File: src/App.js (hunk 7 above)
Comment (luchogabutti) [reply src/App.js]: done

**Review by noelsp — APPROVED** [review]

**Pull request merged**